tracim/tests/functional/test_comments.py

```diff
@@ -0,0 +1,284 @@
+    def test_api__delete_content_comment__ok_200__workspace_manager_owner(self) -> None:
+        """
+        delete comment (user is workspace_manager and owner)
+        """
+        self.testapp.authorization = (
+            'Basic',
+            (
+                'admin@admin.admin',
+                'admin@admin.admin'
+            )
+        )
+        res = self.testapp.get('/api/v2/workspaces/2/contents/7/comments', status=200)
+        assert len(res.json_body) == 3
+        comment = res.json_body[0]
+        assert comment['content_id'] == 18
+        assert comment['parent_id'] == 7
+        assert comment['raw_content'] == '<p> What is for you the best cake ever ? </br> I personnally vote for Chocolate cupcake !</p>'   # nopep8
+        assert comment['author']
+        assert comment['author']['user_id'] == 1
+        # TODO - G.M - 2018-06-172 - [avatar] setup avatar url
+        assert comment['author']['avatar_url'] is None
+        assert comment['author']['public_name'] == 'Global manager'
+        # TODO - G.M - 2018-06-179 - better check for datetime
+        assert comment['created']
+
+        res = self.testapp.delete(
+            '/api/v2/workspaces/2/contents/7/comments/18',
+            status=204
+        )
+        res = self.testapp.get('/api/v2/workspaces/2/contents/7/comments', status=200)
+        assert len(res.json_body) == 2
+        assert not [content for content in res.json_body if content['content_id'] == 18]  # nopep8
+
+    def test_api__delete_content_comment__ok_200__workspace_manager(self) -> None:
+        """
+        delete comment (user is workspace_manager)
+        """
+        self.testapp.authorization = (
+            'Basic',
+            (
+                'admin@admin.admin',
+                'admin@admin.admin'
+            )
+        )
+        res = self.testapp.get('/api/v2/workspaces/2/contents/7/comments', status=200)
+        assert len(res.json_body) == 3
+        comment = res.json_body[1]
+        assert comment['content_id'] == 19
+        assert comment['parent_id'] == 7
+        assert comment['raw_content'] == '<p>What about Apple Pie ? There are Awesome !</p>'   # nopep8
+        assert comment['author']
+        assert comment['author']['user_id'] == 3
+        # TODO - G.M - 2018-06-172 - [avatar] setup avatar url
+        assert comment['author']['avatar_url'] is None
+        assert comment['author']['public_name'] == 'Bob i.'
+        # TODO - G.M - 2018-06-179 - better check for datetime
+        assert comment['created']
+
+        res = self.testapp.delete(
+            '/api/v2/workspaces/2/contents/7/comments/19',
+            status=204
+        )
+        res = self.testapp.get('/api/v2/workspaces/2/contents/7/comments', status=200)
+        assert len(res.json_body) == 2
+        assert not [content for content in res.json_body if content['content_id'] == 19]  # nopep8
+
+    def test_api__delete_content_comment__ok_200__content_manager_owner(self) -> None:
```

Comment:
probably easier to understand `test_api__delete_content_comment__ok_200__content_manager_owner` if named `test_api__delete_comment__ok_200__owner_is_a_content_manager`

Same remark for all test names

Reply:
not really the same in fact. We don't care about if owner_is_a_content_manager, what is really important is that user of the "delete comment" is owner and content_manager.